composer.json

```diff
@@ -45,6 +45,6 @@
     "fabpot/goutte": "^3.2.3",
     "phpunit/phpunit": "^7.3",
     "symfony/css-selector": "^3.4 || ^4.0",
-    "symfony/framework-bundle": "^3.4 || ^4.0"
+    "symfony/symfony": "^3.4 || ^4.0"
```

Comment:
Depending of `symfony/symfony` is considered a bad practice for libraries (since a very long time), this cannot be merged as is.